utils/Rector/RemoveErrorSuppressInTryCatchStmtsRector.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Utils\Rector;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\ErrorSuppress;
+use PhpParser\Node\Stmt\TryCatch;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+final class RemoveErrorSuppressInTryCatchStmtsRector extends AbstractRector
+{
+	public function getRuleDefinition(): RuleDefinition
+	{
+		return new RuleDefinition('Remove error suppress @', [
+			new CodeSample(
+				<<<'CODE_SAMPLE'
+try {
+	@rmdir($dirname);
+} catch (Exception $e) {}
+CODE_SAMPLE
+,
+				<<<'CODE_SAMPLE'
+try {
+	rmdir($dirname);
+} catch (Exception $e) {}
+CODE_SAMPLE
+			),
+		]);
+	}
+
+	/**
+	 * @return string[]
+	 */
+	public function getNodeTypes(): array
+	{
+		return [ErrorSuppress::class];
+	}
+
+	/**
+	 * @param ErrorSuppress $node
+	 */
+	public function refactor(Node $node): ?Node
+	{
+		// not in try catch
+		$tryCatch = $this->betterNodeFinder->findParentType($node, TryCatch::class);
```

Comment:
Will the parent type be also matched to `TryCatch` if the `@` is inside a control structure like `if`?
```php
try {
    if (true) {
        @rmdir('/path');
    }
} catch (\Throwable) {}

```

Reply:
Yes 
![Screen Shot 2021-05-22 at 21 13 34](https://user-images.githubusercontent.com/459648/119229512-965f5d80-bb42-11eb-8645-24d69d4b876b.png)